MessengerProj/src/main/res/values/strings.xml

```diff
@@ -204,11 +204,11 @@
     <!--permissions-->
     <string name="PermissionContacts">Access to contacts needed to connect with your friends across all your devices.</string>
     <string name="PermissionStorage">Storage access needed to send and save photos, videos, music, and other media.</string>
-    <string name="PermissionNoAudio">Microphone access needed to record voice messages.</string>
+    <string name="PermissionNoAudio">Mic access needed to record voice messages.</string>
     <string name="PermissionOpenSettings">Settings</string>
     <!--Intro view-->
     <string name="Intro1Headline">Delta Chat</string>
-    <string name="Intro1Message">The messenger with the <![CDATA[<b>broadest audience</b>]]> in the world. <![CDATA[<b>Free</b>]]> and <![CDATA[<b>independent</b>]]>.</string>
+    <string name="Intro1Message">The messenger with the <![CDATA[<b>broadest audience</b>]]> in the world. <![CDATA[<b>Libre</b>]]> and <![CDATA[<b>independent</b>]]>.</string>
```

Comment:
i would leave this sentence untouched, for now,  this would require a broader discussion. "Free" is unprecise, of course, however, much more known as "libre".

Reply:
Guessing you meant "than libre", which is true, it is also true equal capacity of "free" being more-so a price concern to the lesser known "free software.

Reply:
Maybe use a known term even partly understandable without full knowledge. `Fully public, independent Free Software.`?

Reply:
might be.

but, as mentioned, i would prefer a broader discussion on this point. i would leave it "as is" for now and would be happy to merge the other changes. also i am not really free to dive in here just now. feel free to create another pr with this change or an issue so that it won't get forgotten.